iree/compiler/Conversion/LinalgToSPIRV/LinalgTileAndFusePass.cpp

```diff
@@ -132,7 +132,7 @@ LogicalResult TileSizeCalculator::inferTileAndWorkgroupSize(
     Operation *op = linalgOp.getOperation();
     if (isa<linalg::ConvOp>(op))
       opInfo |= OpInfo::Convolution;
-    else if (isa<linalg::MatmulOp>(op))
+    else if (isa<linalg::MatmulOp, linalg::BatchMatmulOp>(op))
```

Comment:
Not sure the tiling parameters for matmul work exactly for batched matmul. What is the `iterator_types` for the generated tc op? The batch loops need to be accounted for....